.github/scripts/trailing_whitespace_check.sh

```diff
@@ -5,7 +5,7 @@ set -e
 # Add lines "-o <condition> -prune" to add more exceptions
 files="$(find . \
   -type d \( -path ./.git \) -prune \
-  -o -type f \( -name "*.jpg" -o -name "*.png" \) -prune \
+  -o -type f \( -name "*.jpg" -o -name "*.png" -o -name "*.md" -o -name "*.html" \) -prune \
```

Comment:
Just remove trailing white space from the files that you modified. There are some extra spaces in the doc.

Also, you can configure Intellij to do this automatically: https://stackoverflow.com/a/20683165/1775120